platforms/azure/giantnetes/variables.tf

```diff
@@ -76,7 +76,7 @@ variable "master_count" {
 variable "worker_count" {
   type        = string
   description = "Number of worker nodes to be created."
-  default     = "3"
+  default     = "4"
```

Comment:
why did we change this? To comply with AWS?

Reply:
removed. I've used it for few installations where 3 nodes is not enough. But didn't want to merge as I hope autoscaler will solve this on its own